src/Forms/__stories__/FormRadioGroup.stories.js

```diff
@@ -3,20 +3,40 @@ import FormRadioItem from '../FormRadioItem';
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 import {
+    boolean,
+    select,
     withKnobs
 } from '@storybook/addon-knobs';
 
+
+const createRadioGroupProps = (overrides) => ({
+    inline: boolean('inline', false),
+    ...overrides
+});
+
+const createRadioProps = (overrides) => ({
+    disabled: boolean('disabled', false),
+    state: select('Validation State', {
+        'default': null,
+        'valid': 'valid',
+        'invalid': 'invalid',
+        'information': 'information',
+        'warning': 'warning'
```

Comment:
How about we just make validation state in a story util file so we can import it in all these different places.  Makes it easier to update all the spots when a new validationState comes along.
```js
// some-storybook-util-file.js
export const validationStateKnob = select('Validation State', {
  'default': null,
  'valid': 'valid',
  'invalid': 'invalid',
  'information': 'information',
  'warning': 'warning'
});
```

Reply:
That's a good idea! I would prefer to leave it as a followup for when we convert these stories to the new syntax though - I didn't want to do that here and create such a huge diff.